Formula/hyeong.rb

```diff
@@ -0,0 +1,15 @@
+class Hyeong < Formula
+  desc "Hyeo-ung Programming Language Compiler in Rust (beta release)"
```

Comment:
We don't accept beta software, please read https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
Oh, I forgot to remove the beta in description. It was released in stable a while ago.